src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py

```diff
@@ -364,7 +120,8 @@ def readJsonInput(self, test):
             # this is to avoid loading very large JSON in memory
             log.debug(" Read JSON Section: " + test)
             jInput = ""
-            with open(self.jsonFile, 'rb') as f:
+            test_file = self.testFile[test]
```

Comment:
We have already loaded the entire config before. So using ijson now will not have much benefit.